hips/tiles/__init__.py

```diff
@@ -0,0 +1,3 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+"""HiPS description class -- provides methods for parsing the HiPS file properties."""
```

Comment:
This is not the appropriate description of the `hips.tiles` package. It will contain many classes.
Suggest to remove this docstring here, or to replace it with something like
> Classes and functions to manage HiPS tiles.